### DISCUSSION TITLE
What are all the 8-bit GPU registers?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#78702] [Programming Help]
### Body

Hi,

I was going to remake Roller Coaster Tycoon in my Lubuntu VM and it has 16 MB of Video Memory and I was wondering what all of the 8-bit registers are. 
If possible, can you please give me a link?.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by anirudh-hegde** [comment]
Hey, 8-bit registers were common in older hardware, they're less relevant in modern systems like Lubuntu VMs. Linking them directly to your VM's video memory capacity might not be accurate or helpful.

Here are some of the links:
https://eecs.blog/8-bit-computer-registers/
https://en.wikipedia.org/wiki/8-bit_computing
https://www.eecg.utoronto.ca/~amza/www.mindsec.com/files/x86regs.html
https://askubuntu.com/questions/587083/virtualbox-how-to-increase-video-memory

I hope this answer helps.

**Comment by ghost** [comment]
Thank you!
